main.go

```diff
@@ -314,7 +335,7 @@ func runTool(files []string, vaultID *string, nonceOverride *int, quorumOverride
 					}
 				}
 			} else {
-				fmt.Printf("Processing legacy vault \"%s\" (%s).\n", clearVaults[vID].Name, vID)
+				// fmt.Printf("Processing legacy vault \"%s\" (%s).\n", clearVaults[vID].Name, vID)
```

Comment:
@notatestuser not sure what to do here, this would be printed on the first runTool loop

Reply:
it's unnecessary

Reply:
but perhaps we could write a file log that contains info like this, in case we have to do customer support.

Reply:
is the file log something we want in this PR?

Reply:
Not this one